internal/aws/client.go

```diff
@@ -33,6 +33,7 @@ type AWSConfig struct {
 	RecordHostedZoneID string `yaml:"recordHostedZoneID"`
 	DNSName            string `yaml:"dnsName"`
 	AliasHostedZoneID  string `yaml:"aliasHostedZoneID"`
+	Cluster            string `yaml:"cluster"`
```

Comment:
Can you update the variable name here as well to be RoleName please (and in the template), to more correctly reflect what should be passed in.